### ISSUE TITLE
Avoid creating a folder for TTNT's data

### ISSUE THREAD

**Issue by robin850** [Genki-S/ttnt#21]
Hello,

I took the liberty to open an issue to discuss an eventual way to avoid creating a folder for storing TTNT's data. Actually, instead of creating a file for each information, we could rather create a single `.ttnt` file that would store both. For example:

~~~javascript
// Anchor commit: 8b4bac9f164b6389d657f4c1de100dccfc245651
{"test/...":{"lib/..": [1, 2]}}
~~~

Thus you can still call `JSON.load` with the whole file content but we have all information inside a single file. The advantage is also that we are more consistent with other existing specific files (like `.travis.yml` or `.gitignore`).

What do you think ?

**Comment by Genki-S** [comment]
Hi, thank you for your suggestion!

I love your suggestion :heart: Actually, the best part for me is that I can remove `.ttnt/commit_obj.txt` file, whose name I really dislike :disappointed_relieved: 

But I think commenting out meta data like `// ...` is a little hackish since JSON does not have commenting format (though `JSON.parse` is smart enough to ignore that).
Storing meta data as JSON object might be another way to go, like:

```
{ "meta" : { "anchor_commit" :  "..." },
  "mapping" : {"test/...": {"lib/.." : [1, 2]}}}
```

I have no preference as long as `JSON.parse` does what it's expected to do on supported versions. What do you think?

**Comment by robin850** [comment]
> JSON does not have commenting format

Whoah, I didn't know that! :+1: Then we should rather go with your approach ; the file may be consumed by another program for any reason (like a web interface or whatever). Also this is less hack-ish and more modular as we could add other information under the `meta` node later.

**Comment by Genki-S** [comment]
Got it! I'll start implementing that way :+1:

**Comment by Genki-S** [comment]
Implemented in #25 ! Now it's much cleaner :wink:
